source/includes/authenticated_api/_mentor.md.erb

```diff
@@ -0,0 +1,40 @@
+### Assign Mentor
+
+> PUT request body
+
+```json
+{
+  "mentor": {
+    "email": "suzie@example.com"
+  }
+}
+```
+
+> PUT request response
+
+```json
+{
+  "status": "success",
+  "data": {
+    "id": 123,
```

Comment:
Should this mini representation of the user include the email address? The member_id? 

The documentation should at least define what the`id` here is.